test/project_types/script/script_project_test.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'project_types/script/test_helper'
+
+module Script
+  class ScriptProjectTest < MiniTest::Test
+    def setup
+      @context = TestHelpers::FakeContext.new
+      @script_name = 'name'
+    end
+
+    def test_cleanup
```

Comment:
One last nitpick: it would be best if the names of the unit tests reflect what it's doing.  Someone reading through the unit test names should almost feel like reading the spec of that class.  I would suggest the names of these two tests be something like: "test_cleanup_when_directory_exists", and "test_cleanup_when_directory_does_not_exist".